### ISSUE TITLE
JSON5 vulnerability

### ISSUE THREAD

**Issue by yeikel** [peter-evans/find-comment#112]
Hi team, 

Currently, we have the following vulnerability, can we look into fixing it? 

```json
[

  {
    "createdAt": "2023-02-02T17:50:38Z",
    "dismissedAt": null,
    "securityVulnerability": {
      "package": {
        "name": "json5"
      },
      "severity": "HIGH"
    }
  }
]
```

It is possible that https://github.com/peter-evans/find-comment/pull/97 fixed it, but we need a new release

**Comment by yeikel** [comment]
It is possible that https://github.com/peter-evans/find-comment/pull/97 fixed it, but we need a new release

**Comment by peter-evans** [comment]
Hi @yeikel 

Released `v2.2.1` / `v2` with the fix.
